swagger.yaml

```
@@ -572,11 +561,12 @@ paths:
       tags:
       - "Public"
       summary: "Get observations for a census dataset"
-      description: "Get observations for a census dataset, with capability to flex and filter the data to the users need.  If no geography, dimension, or options parameters are supplied then observations for the default geography are returned."
+      description: "Get observations for a census dataset, with capability to flex and filter the data to the users need.  If no area-type parameters are supplied then observations for the default area-type are returned."
```

Can we take the opportunity to remove the extra whitespace?
```suggestion
      description: "Get observations for a census dataset, with capability to flex and filter the data to the users need. If no area-type parameters are supplied then observations for the default area-type are returned."
```

---

This has now been removed.